frame/chainbridge/src/tests.rs

```diff
@@ -183,6 +185,34 @@ fn asset_transfer_success() {
     })
 }
 
+#[test]
+fn asset_transfer_invalid_resource_id() {
+    new_test_ext().execute_with(|| {
+        let dest_id = 2;
+        let to = vec![2];
+        let resource_id = [1; 32];
+        let amount = 100;
+        assert_ok!(Bridge::set_threshold(Origin::root(), TEST_THRESHOLD,));
```

Comment:
Add a newline between the arrange and act/assert sections.

Reply:
Group bridge set up (set_threshold, whitelist_chain) into 1 section without newlines.

Reply:
Updated the code.